recipes/biscuit/meta.yaml

```diff
@@ -1,19 +1,21 @@
-{% set version = "1.2.1.20230601" %}
+{% set version = "1.3.0.20231027" %}
 
 package:
   name: biscuit
   version: {{ version }}
 
 build:
-  number: 2
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('biscuit', max_pin='x') }}
```

Comment:
The versioning is a bit mixed, probably use `x.x` might be better, i.e.
`- {{ pin_subpackage('biscuit', max_pin='x.x') }}`

Reply:
@thanhleviet I updated the versioning as requested and all checks are passing. Let me know if there are further updates needed.